worlds/banjo_tooie/__init__.py

```diff
@@ -0,0 +1,1094 @@
+                    item = self.create_item(itemName.GIA)
+                elif world == regionName.JR:
+                    item = self.create_item(itemName.JRA)
+                else:
+                    item = self.create_item(world)
+                self.get_location(f"World {world_num} Unlocked").place_locked_item(item)
+                world_num += 1
+
+        if self.options.victory_condition == VictoryCondition.option_minigame_hunt\
+                or self.options.victory_condition == VictoryCondition.option_wonderwing_challenge:
+
+            item = self.create_item(itemName.MUMBOTOKEN)
+            for location_name in MumboTokenGames_table.keys():
+                self.get_location(location_name).place_locked_item(item)
+
+        if self.options.victory_condition == VictoryCondition.option_boss_hunt \
+                or self.options.victory_condition == VictoryCondition.option_wonderwing_challenge \
+                or self.options.victory_condition == VictoryCondition.option_boss_hunt_and_hag1:
+
+            item = self.create_item(itemName.MUMBOTOKEN)
+            for location_name in MumboTokenBoss_table.keys():
+                self.get_location(location_name).place_locked_item(item)
+
+        if self.options.victory_condition == VictoryCondition.option_jinjo_family_rescue\
+                or self.options.victory_condition == VictoryCondition.option_wonderwing_challenge:
+            item = self.create_item(itemName.MUMBOTOKEN)
+            for location_name in MumboTokenJinjo_table.keys():
+                self.get_location(location_name).place_locked_item(item)
+
+        if not self.options.randomize_jinjos:
+            self.get_location(locationName.JIGGYIH1).place_locked_item(self.create_item(itemName.JIGGY))
+            self.get_location(locationName.JIGGYIH2).place_locked_item(self.create_item(itemName.JIGGY))
+            self.get_location(locationName.JIGGYIH3).place_locked_item(self.create_item(itemName.JIGGY))
+            self.get_location(locationName.JIGGYIH4).place_locked_item(self.create_item(itemName.JIGGY))
+            self.get_location(locationName.JIGGYIH5).place_locked_item(self.create_item(itemName.JIGGY))
+            self.get_location(locationName.JIGGYIH6).place_locked_item(self.create_item(itemName.JIGGY))
+            self.get_location(locationName.JIGGYIH7).place_locked_item(self.create_item(itemName.JIGGY))
+            self.get_location(locationName.JIGGYIH8).place_locked_item(self.create_item(itemName.JIGGY))
+            self.get_location(locationName.JIGGYIH9).place_locked_item(self.create_item(itemName.JIGGY))
+
+            item = self.create_item(itemName.WJINJO)
+            self.get_location(locationName.JINJOJR5).place_locked_item(item)
+
+            item = self.create_item(itemName.OJINJO)
+            self.get_location(locationName.JINJOWW4).place_locked_item(item)
+            self.get_location(locationName.JINJOHP2).place_locked_item(item)
```

Comment:
```suggestion
            locations = [locationName.JINJOWW4, locationName.JINJOHP2]
            for loc in locations:
                self.get_location(loc).place_locked_item(self.create_item(itemName.OJINJO))
```
Not a big deal for the two here, but I feel like it'd be better for these to be replaced by a loop similar to this. Also I'm not sure if matters for locked items, but I feel like it's best to avoid re-using the same item instance and instead create a new one like for Jiggys here.